akka-http-core/src/main/resources/reference.conf

```diff
@@ -206,6 +206,21 @@ akka.http {
       # be increased for high bandwidth-delay-product connections.
       incoming-stream-level-buffer-size = 512kB
     }
+
+    websocket {
+      # periodic keep alive may be implemented using by sending Ping frames
+      # upon which the other side is expected to reply with a Pong frame,
+      # or by sending a Pong frame, which serves as unidirectional heartbeat.
+      # Valid values:
+      #   ping - default, for bi-directional ping/pong keep-alive heartbeating
+      #   pong - for uni-directional pong keep-alive heartbeating
+      periodic-keep-alive-mode = ping
```

Comment:
Maybe mention here somewhere also that you can programmatically provide custom ping-pong payload?